agg-svn/agg-2.4/include/agg_rasterizer_cells_aa.h

```diff
@@ -185,9 +185,6 @@ namespace agg
         {
             if((m_num_cells & cell_block_mask) == 0)
             {
-                if(m_num_blocks >= m_cell_block_limit) {
-                    throw std::overflow_error("Exceeded cell block limit");
-                }
                 allocate_block();
```

Comment:
This check needs to be inside `allocate_block`. Otherwise the rasterizer instance becomes unusable after hitting the cell block limit.